src/bidiMapper/domains/events/SubscriptionManager.ts

```diff
@@ -130,7 +130,26 @@ export class SubscriptionManager {
 
     // Get all the subscription priorities.
     const priorities: number[] = relevantContexts
-      .map((c) => contextToEventMap.get(c)?.get(eventMethod))
+      .map((context) => {
+        // Get the priority for exact event name
+        const priority = contextToEventMap.get(context)?.get(eventMethod);
+        // For CDP we can't provide specific event name when subscribing
+        // to the module directly.
+        // Because of that we need to see event `cdp` exits in the map.
+        if (eventMethod.startsWith(ChromiumBidi.BiDiModule.Cdp)) {
+          const cdpPriority = contextToEventMap
+            .get(context)
+            ?.get(ChromiumBidi.BiDiModule.Cdp);
+          // If we subscribe to the event directly and `cdp` module as well
+          // priority will be different we take the one with higher priority
```

Comment:
The comment says "take the one with higher priority", but we're using `Math.min`. Is this intentional?

Reply:
It should be the same as the other comment `minimum` priority.